README.md

```diff
@@ -7,7 +7,7 @@
 ## Dive in
 
 * [Documentation](https://adaptivecards.io/documentation/)
-* [Roadmap](https://portal.productboard.com/adaptivecards/1-adaptive-cards-features/tabs/5-launched)
+* [Roadmap](https://github.com/microsoft/AdaptiveCards/projects/32)
```

Comment:
the above link doesn't work for me.

Reply:
works for me :)

Reply:
tested without a github account and also worked meaning it is accessible to public.